frontend/src/components/Footer/Footer.stories.tsx

```diff
@@ -2,46 +2,57 @@ import { Meta, Story } from '@storybook/react'
 
 import { viewports } from '~utils/storybook'
 
-import { Footer, FooterProps } from './Footer'
+import { FooterProps } from './common/types'
+import { Footer } from './Footer'
 
-export default {
-  title: 'Components/Footer',
-  component: Footer,
-  decorators: [],
-  parameters: {
-    layout: 'fullscreen',
-  },
-} as Meta
-
-const defaultArgs: FooterProps = {
+const DEFAULT_ARGS: FooterProps = {
   appName: 'Form',
-  tagline: 'Build secure government forms in minutes',
   footerLinks: [
     {
-      label: 'User Guide',
+      label: 'User guide',
```

Comment:
would it make sense to import this from the app rather than keeping it in two places? to make sure storybook reflects what's on the app

Reply:
This is the general Footer story, not actually the app's footer. Can be kept separate bah